app/openWindow.js

```diff
@@ -104,6 +104,10 @@ const openWindow = ( store ) =>
         {
             browserWindowArray.splice( index, 1 );
         }
+        if ( process.platform !== 'darwin' )
+        {
+            app.quit();
```

Comment:
We need to check the window count before doing quit. 

Otherwise this is working 👍